### PR TITLE
update single cell instance home text

### PR DESCRIPTION
**Pull request by nomadscientist**
Have updated single cell text and reformatted minor things

### CODE REVIEW

**Comment by nomadscientist** [comment]
Full disclosure idk why but when I cloned this repo it kept saying I'd changed the community event update from 2022 and I can't for the life of me make it stop. I don't care about those files, ignore them, dismiss those changes, whatever. I just want the text updated on the single cell page :)

File: index-singlecell.md [hunk 1]
```diff
@@ -28,27 +26,35 @@ The server is part if the European Galaxy server and is maintained by the [RNA B
 {:toc}
 
 
-# Get started with
+# Get started with #singlecell
```
Comment (bgruening) [inline index-singlecell.md:29]: Is the second `#` needed?

File: index-singlecell.md [hunk 2]
```diff
@@ -76,29 +84,5 @@ Want to talk with trainers and developers on how to improve single cell analysis
...
...y01) </li><li>[Andrey Solovyev](https://github.com/a-solovyev12) </li><li>[Bérénice Batut](https://github.com/bebatut) </li><li>[Bjoern Gruening](https://github.com/bgruening) </li><li>[Carlos Talavera-Lopez](https://github.com/cartal) </li><li>[Charles Girardot](https://github.com/cgirardot) </li><li>[Christophe Antoniewski](https://github.com/drosofff) </li><li>[Daniel Blankenberg](https://github.com/blankenberg) | <ul><li>[Fidel Ramirez](https://github.com/fidelram) </li><li>[Graham Etherington](https://github.com/ethering) </li><li>[Helena Rasche](https://training.galaxyproject.org/training-material/hall-of-fame/hexylena/) </li><li>[Irene Papatheodorou](https://twitter.com/irenepapatheodo) </li><li>[James Taylor](https://github.com/jxtx) </li><li>[Jelle Scholtalbers](https://github.com/scholtalbers) </li><li>[Jonathan Manning](https://github.com/pinin4fjords) </li><li>[Krzysztof Polanski](https://github.com/ktpolanski) </li><li>[Lea Bellenger](https://github.com/bellenger-l) </li><li>[Maria Doyle](https://github.com/mblue9) </li><li>[Ni Huang](https://github.com/nh3) </li><li>[Nicola Soranzo](https://github.com/nsoranzo) </li><li>[Suhaib Mohammed](https://github.com/suhaibMo) |
```
Comment (bgruening) [inline index-singlecell.md:87]: why is this change needed?

Comment (nomadscientist) [reply index-singlecell.md:87]: Table looks better than the absurdly giant contributors list

File: index-singlecell.md (hunk 1 above)
Comment (nomadscientist) [reply index-singlecell.md:29]: Well it's an interesting point
The first # just makes it a title but the second one is to subtly ingrain into users heads the idea of tagging stuff with #singlecell so we eventually (dreaming longterm) end up with a wealth of useful workflows shared by users all appropriately tagged and thus searchable with #singlecell

Comment (bgruening) [reply index-singlecell.md:29]: mh, ok, we merge quickly and you can iterate - its your site overall :)
